src/test/java/pages/NovusPage.java

```diff
@@ -0,0 +1,53 @@
+package pages;
+
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+import common.BasePage;
+
+import static com.codeborne.selenide.CollectionCondition.sizeGreaterThan;
+import static com.codeborne.selenide.Condition.interactable;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.*;
+
+public class NovusPage extends BasePage {
+
+    private static final SelenideElement NOVUS_MARKET = $x("//a[@href='https://novus.zakaz.ua/uk/']");
+    private static final SelenideElement OFFERS_OF_MARKET = $("[data-marker='offers widget']");
```

Comment:
Change locator to xpath 
```suggestion
    private static final SelenideElement MARKET_OFFERS = $x("//a[@data-marker='offers widget']");
```